regolith/builder.py

```diff
@@ -14,6 +15,7 @@
     'grade': GradeReportBuilder,
     'grades': GradeReportBuilder,
     'resume': ResumeBuilder,
+    'cp': CPBuilder
```

Comment:
I would also add a more descriptive name to this dict, such as 'current-pending'